GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -637,6 +637,21 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         }
                     }
                     //
+                    EffectGUIDEvent orbExploded = log.CombatData.GetEffectGUIDEvent(EffectGUIDs.HarvestTempleOrbExplosion);
+                    if (orbExploded != null)
+                    {
+                        IReadOnlyList<EffectEvent> orbEffects = log.CombatData.GetEffectEventsByEffectID(orbExploded.ContentID);
```

Comment:
this should be limited to the aware interval of the orb agent (like the effects above) otherwise you are going to add each explosions for every orbs